Makefile

```diff
@@ -135,6 +143,30 @@ local-cross: bin/skopeo.darwin.amd64 bin/skopeo.linux.arm bin/skopeo.linux.arm64
 build-container:
 	${CONTAINER_RUNTIME} build ${BUILD_ARGS} -t "$(IMAGE)" .
 
+# Build container specific to arch
+build-arch-container:
+	${CONTAINER_RUNTIME} build -t "${ARCH_IMAGE}" contrib/skopeoimage/${SOURCE_TYPE}
```

Comment:
The name suggests that this is related to `build-container` above, which is very different (this creates a container for users to consume, the above creates a private utility container used for in-container builds).

I don’t have a good suggestion. `user-container:`? Or just write good comments for both of the targets? (“Build container specific to arch” does not say much that the target name doesn’t imply already)

Reply:
hmmm, ok, will think how to cover in comments the differrence. And yes, you are right, names should highlight this difference

Reply:
now separate Makefile for Travis build